app/requirements.txt

```diff
@@ -74,6 +74,7 @@ pytest==8.0.2
     #   pytest-django
 pytest-cov==4.1.0
 pytest-django==4.8.0
+pytest-xdist==3.6.1
```

Comment:
You should put this line in `app/requirements.in` and generate the `requirements.txt` from that. No need to specify a version in the *.in file. It'll write the latest version in the *.txt file.